pf/internal/plugin/provider_server.go

```diff
@@ -80,5 +81,41 @@ func (p providerThunk) Configure(
 	if ctx.Value(setupConfigureKey) != nil {
 		return plugin.ConfigureResponse{}, nil
 	}
+	req.Inputs = removeSecrets(req.Inputs)
```

Comment:
I'm confused here, is the panic we're trying to fix scoped to PF, or SDKv2, or both?

Reply:
Just PF (and muxed providers that use PF).